step-release-vis/src/app/services/environment_test.ts

```diff
@@ -2,6 +2,13 @@ import { TestBed } from '@angular/core/testing';
 
 import { EnvironmentService } from './environment';
 import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { CandidateInfo } from '../models/Data';
+
+function fractionalEqual(x: number, y: number): boolean {
```

Comment:
Remove this if not needed :)

Reply:
sorry, forgot about that. done in new commit